docs/api/graphql_queries.md

```diff
@@ -181,6 +181,72 @@ Response:
 }
 ```
 
+### Get Matrix Field Type
```

Comment:
Shouldn't this rather be in the fieldtype reference part, where we also document how to get a field data using PHP and Twig, and for some using REST (Rich Text) ? It also feels odd that only Matrix is referenced in this particular doc. A link to the fieldtype reference chapter would be more logical imho.